build/CMakeCache.txt

```diff
@@ -0,0 +1,338 @@
+# This is the CMakeCache file.
```

Comment:
Are these Cmake and build files supposed to be in here? These don't seem to be needed for the PR

Reply:
No they arent, im not sure how they got added im going to remove them